app/src/validators/user.js

```diff
@@ -10,7 +10,7 @@ const schema = {
       identityId: scheme.string,
       idp: scheme.string,
       username: type.alphanum,
-      email: type.email,
+      email: Joi.string(),
```

Comment:
Is `type.email` not working? `createInvite` in `validators/invite.js` is also using the former.

Reply:
_Note:_ elevated users can search on partial email matches, while non-elevated users require an exact match, so we'll have to accept any string unless we decide to implement more complex validator logic.